src/main/java/digital/slovensko/autogram/core/Updater.java

```diff
@@ -38,10 +40,8 @@ public static boolean newVersionAvailable() {
             return false;
         }
 
-        if (latestVersionTag.equals(""))
-            return false;
-
-        var latestVersion = latestVersionTag.replaceAll("^v", "");
-        return !Main.getVersion().equals(latestVersion);
+        Version vCurrent = new Version(Main.getVersion());
```

Comment:
Main.getVersion by mohla rovno vratit version len treba osetrit specialny pripad s dev.

Reply:
to je trochu neprakticke, kedze sa pouziva aj na viacerych miestach ako string a do appky prichadza ako string

Reply:
Do appky ze to pride ako string rozumiem, ale potom uz mi to pride ako https://wiki.c2.com/?PrimitiveObsession

Reply:
no urobil som to, ale pride mi to zbytocne komplikovane (vid posledny commit)